mysqld_exporter.go

```diff
@@ -170,6 +172,74 @@ func init() {
 	prometheus.MustRegister(version.NewCollector("mysqld_exporter"))
 }
 
+func filter(filters map[string]bool, name string) bool {
+	f := flag.Lookup("collect." + name)
```

Comment:
The only down-side to this approach is that we want to switch to [kingpin](https://github.com/alecthomas/kingpin) for flag parsing.  I haven't been able to find an equivalent feature in kingpin. 😦

Reply:
Yes, it would have be nice if kingpin had a `Lookup()` method 😕
I can change the `filter()` implementation or submit a PR for kingpin 😅

Reply:
Oh, I totally forgot to merge https://github.com/prometheus/mysqld_exporter/pull/222, which does the kingpin change.  I would like to merge that, and have the filter changed to match.  I think it will take too much time to get it implemented in kingpin.

Sorry, I should have taken care of the flag issue first.

Reply:
No problem, I'll update my branch.